src/bundler.ts

```diff
@@ -387,11 +395,34 @@ class Bundler {
    * Given a URL to an entry-point html document, produce a single document
    * with HTML imports, external stylesheets and external scripts inlined,
    * according to the options for this Bundler.
+   *
+   * TODO: Given Multiple urls, produces a sharded build by applying the provided
+   * strategy.
+   * 
+   * @param {Array<string>} entrypoints The list of entrypoints that will be
+   *     analyzed for dependencies. The results of the analysis will be passed
+   *     to the `strategy`. An array of length 1 will bypass the strategy and
+   *     directly bundle the document.
+   * @param {BundleStrategy} strategy The strategy used to construct the
+   *     output bundles. See 'polymer-analyzer/lib/bundle-manifest' for
+   *     examples. UNUSED.
    */
-  async bundle(url: string): Promise<DocumentCollection> {
+  async bundle(entrypoints: string[], strategy?: BundleStrategy):
```

Comment:
Until we support multiple entrypoints for realsies maybe a `console.assert` that its length === 1?